Build/CIRCLECI_SETUP.md

```diff
@@ -0,0 +1,246 @@
+# CircleCI variables setup
+
+This is how to make it build inside CircleCI. The licenses are needed to be
+added via environment variables inside the project settings within CircleCI.
+
+## Setup Unity licenses
+
+### Obtain Unity license .ulf files
+
+Repeat the following steps for the following versions:
+
+- 2018.4.14f1
+- 2019.2.11f1
+- 2020.1.0b6-linux-il2cpp
+
+1. Run `Build/local_docker_run.ps1` in PowerShell with `-UnityVersion` parameter
+   being set to Unity version, ex:
+
+    ```powershell
+    # Assuming working directory is at repo root
+    &'.\Build\local_docker_run.ps1' -UnityVersion '2019.2.11f1'
+    ```
+
+    If running Linux or MacOS, you can install [PowerShell Core](https://github.com/PowerShell/PowerShell)
+    to run PowerShell scripts, and then invoke the script like so:
+
+    ```bash
+    # Assuming working directory is at repo root
+    pwsh ./Build/local_docker_run.ps1 -UnityVersion '2019.2.11f1'
+    ```
+
+    If using Git Bash for Windows or WSL, you could also run the script like so:
+    ```bash
+    # Assuming working directory is at repo root
+    powershell.exe ./Build/local_docker_run.ps1 -UnityVersion '2019.2.11f1'
+    ```
+
+2. Enter personal username+password
+
+3. Wait for the log to spit out XML
+
+4. Copy that XML
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `1`, was `4`](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461394)